content/usage/secrets.md

```diff
@@ -9,6 +9,8 @@ toc = true
 
 # Overview
 
+> Note: Secrets are injected into the Yaml `build` and `compose` sections by default for pull requests. If you need different behavior for security reasons please see [pull requests(http://readme.drone.io/usage/secrets/#pull-requests:0d943d142dc6bcdd2e544b095a49f72b).
```

Comment:
secrets are also injected into the cache, clone and notify steps for pull requests, with this new change. perhaps we could make this warning a little more generic, like this:

> Warning: drone does not prevent your from inadvertently exposing your secrets to the world. Please read this section of the documentation carefully to ensure you follow best practices and avoid exposing sensitive data.